UserService/app/src/main/java/com/bookmyshow/userservice/service/PermissionService.java

```diff
@@ -0,0 +1,65 @@
+package com.bookmyshow.userservice.service;
+
+import com.bookmyshow.userservice.dao.entity.Permission;
+import com.bookmyshow.userservice.dao.repo.PermissionRepository;
+import com.bookmyshow.userservice.dto.PermissionRequestDTO;
+import com.bookmyshow.userservice.dto.PermissionResponseDTO;
+import com.bookmyshow.userservice.exception.PermissionNotFoundException;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+public class PermissionService {
+
+    @Autowired
+    private ModelMapper modelMapper;
+
+    @Autowired
+    private PermissionRepository repository;
+
+    public PermissionResponseDTO addPermission(PermissionRequestDTO permissionRequestDTO){
```

Comment:
We are already in Permission Service. So we can exclude Permission in the method name. It is good to align this method name as "Save"